appengine/search/src/main/java/com/example/appengine/search/DeleteServlet.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.appengine.search;
+
+// @formatter:off
+// [START delete_import]
```

Comment:
Cool! I was actually wondering how we were going to show imports, since many existing samples do this. We should definitely add this checkstyle stuff to java-repo-tools.